src/reducers/heroes.js

```diff
@@ -42,6 +42,20 @@ const heroes = (state = initialState, action) => {
       delete copy[action.characterId];
       return copy;
     }
+    case FREEZE: {
+      const frozenIds = action.ids;
+      const newFreeze = {};
+
+      frozenIds.forEach((id) => {
+        if (state[id] !== undefined) {
```

Comment:
using hasOwnProperty() seems to cause a lint error.  "no-prototype-builtins"

> Do not access Object.prototype method 'hasOwnProperty' from target object

Unless I've misunderstood how to use it.